### PR TITLE
BUG add libfreebl3.so

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@isuruf @wolfv

**Comment by wolfv** [comment]
Looks great to me! Do you want me to test this PR somehow?

**Comment by beckermr** [comment]
I don' think we need to test it...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -104,7 +108,8 @@ about:
   home: http://sources.redhat.com/glibc/
   license: LGPL-2.0-or-later AND LGPL-2.0-or-later WITH exceptions AND GPL-2.0-or-later
```
Comment (beckermr) [inline recipe/meta.yaml:109]:
```suggestion
  license: LGPL-2.0-or-later AND LGPL-2.0-or-later WITH exceptions AND GPL-2.0-or-later AND MPL-2.0
```

**Pull request merged**